Server/JokesOnYou.Web.Api/Extensions/ServiceExtensions.cs

```diff
@@ -2,16 +2,24 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using JokesOnYou.Web.Api.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace JokesOnYou.Web.Api.Extensions
 {
     public static class ServiceExtensions
     {
-        public static IServiceCollection ConfigureAppServices(this IServiceCollection services)
+        public static IServiceCollection ConfigureAppServices(this IServiceCollection services, IConfiguration config)
         {
+            services.AddDbContext<DataContext>(options =>
+                options.UseSqlite(config.GetConnectionString("DefaultConnection")));
+
+            
```

Comment:
I'm being Pedantic! But we should look at implementing style cop or setting up an editor config to maintain coding standard.

Reply:
Yes that would be good! you could open an issue if you like.
If you know how you can also pick it up ?